documentation/src/docs/auth/client-auth.md

```diff
@@ -0,0 +1,35 @@
+<!--
+Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+# Client Auth interface
+Pravega client can access Pravega APIs through `grpc`. Some of the admin APIs can be accessed via `REST` API. 
+The authorization/authentication API and plugin works for both these interfaces.
```

Comment:
The Authorization/Authentication API and plugin works for both these interfaces.